bootstrap/lib/util.js

```diff
@@ -0,0 +1,46 @@
+const { v4: uuidv4 } = require('uuid')
+
+const parseBool = (value) => {
+  if (!value) return false
+  const _val = value.toString().toLowerCase()
+  return (_val === 'true' || _val === 'false') && _val === 'true'
+}
+
+// We generate an UUID per instance
+const uuid = () => {
+  if (!process.env.UUID) process.env.UUID = uuidv4()
+  return process.env.UUID
+}
+
+const toAsync = (execute, data) =>
+  new Promise((resolve) => execute(data, (statusCode, data) => resolve({ statusCode, data })))
+
+const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms))
+
+const interval = (retryCount = 1, min = 100, max = 1000, coefficient = 2) => {
```

Comment:
I think the following change would make this more clear to read, and is less computationally expensive
```ts
/**
 * Return a value used for exponential backoff in milliseconds.
 * @example
 * exponentialBackoffMs(1,100,1000,2) === 100
 * exponentialBackoffMs(2,100,1000,2) === 200
 * exponentialBackoffMs(3,100,1000,2) === 400
 *
 * @param retryCount The amount of retries that have passed
 * @param min The interval in ms
 * @param max The maximum backoff in ms
 * @param coefficient The base multiplier
 */
function exponentialBackoffMs(retryCount = 1, min = 100, max = 1000, coefficient = 2) {
  return Math.min(max, min * coefficient ** (retryCount - 1))
}
```

Reply:
Love it👌 Thanks!